src/shm_ext.h

```diff
@@ -129,10 +129,11 @@ sr_error_info_t *sr_shmext_change_sub_stop(sr_conn_ctx_t *conn, sr_mod_t *shm_mo
  * @param[in] sub_type Data-provide subscription type.
  * @param[in] sub_opts Subscription options.
  * @param[in] evpipe_num Subscription event pipe number.
+ * @param[in] prio Subscription priority.
```

Comment:
Not an `in` parameter, fix and add description.

Reply:
Done